toughday/src/main/java/com/adobe/qe/toughday/internal/core/engine/PublishMode.java

```diff
@@ -29,10 +29,15 @@ public PublishMode() {
         this.globalRunMap = new RunMapImpl();
     }
 
+    @Deprecated
     public RunMapImpl getGlobalRunMap() {
         return globalRunMap;
     }
 
+    public RunMapImpl getRunMap() {
+        return globalRunMap;
+    }
```

Comment:
Well, i figured the name "getGlobalRunMap()" does not really fit any more, since there is a run map for every test. I could rename the field globalRunMap to runMap so it will make more sense in PublishMode, as well.